docs/src/app/app.jsx

```diff
@@ -59,6 +60,8 @@ const styles = (theme) => ({
     },
 });
 
+console.log(moment.tz.guess());
```

Comment:
Uh oh!  We have a rogue `console.log()`.  Let's remove that please!  And likely the import as well if it's not needed.

Reply:
Oppsi-whoppsi

Reply:
Fixed.

Reply:
Looks good.  Resolving.